### DISCUSSION TITLE
Will consider adding `hidden` attributes to `Button` components?

### DISCUSSION THREAD

**Discussion by vingeraycn** [rsuite/rsuite#1789] [General]
such as:

```tsx
<Button hidden>Something</Button>
```

**Comment by SevenOutman** [comment]
`hidden` is an existing [attribute on HTMLElement](https://developer.mozilla.org/zh-CN/docs/Web/API/HTMLElement/hidden), and `<Button hidden>` is already passing this attribute to the underlying `<button>` element.

If you want to hide the button when passing `hidden=true`, you can simply apply CSS to `[hidden]` selector.

**Reply by vingeraycn** [reply]
`.rs-btn` has overrided the default styles of `[hidden]`
<img width="297" alt="图片" src="https://user-images.githubusercontent.com/19839331/126595241-a48e9307-f3f7-4aa6-9c24-605818eae0e6.png">

**Reply by SevenOutman** [reply]
You may apply `!important` to `[hidden]` if it's supposed so.
